py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/postgres_utils.py

```diff
@@ -192,7 +192,7 @@ def insert_metadata(written_file) -> None:  # type: ignore
                 processed=False, path=filepath
             )
             connection.execute(stmt)
-            connection.commit()
+            connection.commit()  # type: ignore
```

Comment:
mypy was yelling about this.

i moved it to ignore since we should move all this to either a shared lib or remove the ingest lambdas.